packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -21,10 +22,14 @@ import {
 } from '../TransactionsTable/TransactionsTable'
 import { SafeImage } from './SafeImage'
 import { ReactComponent as CustomClipboardCopyIcon } from '../../assets/copy.svg'
+import { ChainId } from 'src/util/networks'
```

Comment:
Let's do an import from relative path here.

Reply:
ah it's not needed. let me remove it!